src/redux/features/taskBoardSlice.tsx

```diff
@@ -57,10 +55,7 @@ const taskBoardSlice = createSlice({
       let taskToUpdate = state.tasks.find((task) => task.id === action.payload.taskId)
       if (taskToUpdate) {
         taskToUpdate.workflowStateId = action.payload.targetWorkflowStateId
-        const updatedTasks = sortTaskByDescendingOrder([
-          ...state.tasks.filter((task) => task.id !== action.payload.taskId),
-          taskToUpdate,
-        ])
+        const updatedTasks = [...state.tasks.filter((task) => task.id !== action.payload.taskId), taskToUpdate]
```

Comment:
I think this makes it so that when a task is updated it gets pushed to the last element of tasks array. Where do we do sorting after this?

Reply:
We are doing sorting in the TaskBoard component for each workflowState columns.